src/main/config/messages_template.xml

```diff
@@ -438,409 +440,409 @@ See the accompanying LICENSE file for applicable license.
   </message>
-  
+
   <message id="DOTX064W" type="WARN">
-    <reason>The copy-to attribute [copy-to="%1"] uses the name of a file that already exists, so this attribute is ignored.</reason>
+    <reason>The @copy-to attribute value '%1' uses the name of a resource that already exists, so this attribute is ignored.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX065W" type="WARN">
-    <reason>Two unique source files each specify copy-to="%2", which results in a collision. The value associated with href="%1" is ignored.</reason>
-    <response></response>
+    <reason>Two unique source files each specify a @copy-to attribute value '%2', which results in a collision.</reason>
+    <response>Ignoring the @copy-to value associated with the @href value '%1'.</response>
   </message>
-  
+
   <message id="DOTX066W" type="WARN">
-    <reason>Template "%1" is deprecated.</reason>
-    <response>Remove references to this template from your custom XSLT or plug-ins.</response>
+    <reason>The '%1' template is deprecated.</reason>
+    <response>Remove references to this template from the custom XSLT or plug-ins.</response>
   </message>
-  
+
   <message id="DOTX067E" type="ERROR">
     <reason>No string named '%1' was found for language '%2'.</reason>
     <response>Add a mapping for the string '%1'.</response>
   </message>
-  
+
   <message id="DOTX068W" type="WARN">
-    <reason>A topicref element that references a map contains child topicref elements.</reason>
-    <response>Child topicref elements are ignored.</response>
+    <reason>A &lt;topicref&gt; element that references a map contains child &lt;topicref&gt; elements.</reason>
+    <response>Ignoring child topic references.</response>
   </message>
-  
+
   <message id="DOTX069W" type="WARN">
-    <reason>Template mode "%1" is deprecated.</reason>
-    <response>Remove references to this template mode from your custom XSLT or plug-ins.</response>
+    <reason>The '%1' template mode is deprecated.</reason>
+    <response>Remove references to this template mode from custom XSLT or plug-ins.</response>
   </message>
-  
+
   <message id="DOTX070W" type="WARN">
-    <reason>Target "%1" is deprecated.</reason>
-    <response>Remove references to this target from your custom Ant files.</response>
-  </message>
-  
-  <message id="DOTX071W" type="WARN">
-    <reason>Parameter "%1" on template "%2" is deprecated.</reason>
-    <response>Use parameter "%3" instead.</response>
+    <reason>The '%1' target is deprecated.</reason>
+    <response>Remove references to this target from custom Ant files.</response>
   </message>
 
   <message id="DOTX071E" type="ERROR">
-    <reason>Conref range: Unable to find conref range end element with ID "%1".</reason>
-    <response/>
+    <reason>Unable to find conref range end element with ID '%1'.</reason>
+    <response></response>
   </message>
-  
+
+  <message id="DOTX071W" type="WARN">
```

Comment:
I cannot find DOTX071W, but I do find the error with same number. I think we can mark the warning one here deprecated